main/src/org/destinationsol/menu/MainScreen.java

```diff
@@ -38,125 +36,120 @@
+        this.isMobile = isMobile;
         this.gameOptions = gameOptions;
 
-        tutCtrl = new SolUiControl(menuLayout.buttonRect(-1, 1), true, Input.Keys.T);
-        tutCtrl.setDisplayName("Tutorial");
-        controls.add(tutCtrl);
+        tutorialControl = new SolUiControl(menuLayout.buttonRect(-1, 1), true, Input.Keys.T);
+        tutorialControl.setDisplayName("Tutorial");
+        controls.add(tutorialControl);
 
-        newGameCtrl = new SolUiControl(menuLayout.buttonRect(-1, 2), true, gameOptions.getKeyShoot());
-        newGameCtrl.setDisplayName("New Game");
-        controls.add(newGameCtrl);
+        newGameControl = new SolUiControl(menuLayout.buttonRect(-1, 2), true, gameOptions.getKeyShoot());
+        newGameControl.setDisplayName("New Game");
+        controls.add(newGameControl);
 
-        optionsCtrl = new SolUiControl(mobile ? null : menuLayout.buttonRect(-1, 3), true, Input.Keys.O);
-        optionsCtrl.setDisplayName("Options");
-        controls.add(optionsCtrl);
+        optionsControl = new SolUiControl(isMobile ? null : menuLayout.buttonRect(-1, 3), true, Input.Keys.O);
+        optionsControl.setDisplayName("Options");
+        controls.add(optionsControl);
 
-        exitCtrl = new SolUiControl(menuLayout.buttonRect(-1, 4), true, gameOptions.getKeyEscape());
-        exitCtrl.setDisplayName("Exit");
-        controls.add(exitCtrl);
+        exitControl = new SolUiControl(menuLayout.buttonRect(-1, 4), true, gameOptions.getKeyEscape());
+        exitControl.setDisplayName("Exit");
+        controls.add(exitControl);
 
-        creditsCtrl = new SolUiControl(creditsBtnRect(r), true, Input.Keys.C);
-        creditsCtrl.setDisplayName("Credits");
-        controls.add(creditsCtrl);
+        creditsControl = new SolUiControl(MenuLayout.bottomRightFloatingButton(resolutionRatio), true, Input.Keys.C);
+        creditsControl.setDisplayName("Credits");
+        controls.add(creditsControl);
 
-        titleLogoTex = textureManager.getTexture("ui/titleLogo");
-        titleBgTex = textureManager.getTexture("ui/titleBg");
-    }
-
-    public static Rectangle creditsBtnRect(float r) {
-        final float CREDITS_BTN_W = .15f;
-        final float CREDITS_BTN_H = .07f;
-
-        return new Rectangle(r - CREDITS_BTN_W, 1 - CREDITS_BTN_H, CREDITS_BTN_W, CREDITS_BTN_H);
+        logoTex = textureManager.getTexture("ui/titleLogo");
+        bgTex = textureManager.getTexture("ui/titleBg");
     }
 
     public List<SolUiControl> getControls() {
         return controls;
     }
 
     @Override
-    public void updateCustom(SolApplication cmp, SolInputManager.Ptr[] ptrs, boolean clickedOutside) {
-        if (cmp.getOptions().controlType == GameOptions.CONTROL_CONTROLLER) {
-            tutCtrl.setEnabled(false);
+    public void updateCustom(SolApplication solApplication, SolInputManager.Ptr[] pointers, boolean clickedOutside) {
```

Comment:
Okay in this case I guess `pointers` makes sense (even if the underlying code probably could use some straightening out)